src/Utilities/hooks/Hooks.test.js

```diff
@@ -94,79 +79,61 @@ describe('Custom hooks tests', () => {
         ];
         let index = 0;
         let direction = SortByDirection.asc;
-        let result = { sort: 'a' };
+        let finalResult = { sort: 'a' };
         let callback = jest.fn();
-        let onSort;
-        testHook(() => {
-            onSort = useSortColumn(columns, callback);
-        });
-        onSort({}, index, direction);
+        const { result } = renderHook(() => useSortColumn(columns, callback));
+        result.current({}, index, direction);
 
-        expect(callback).toHaveBeenCalledWith(result);
+        expect(callback).toHaveBeenCalledWith(finalResult);
     });
     it('usePagePerPage, should return correct page and perPage', () => {
         let limit = 50;
         let offset = 1100;
         let expected = [23, 50];
-        let res;
-        testHook(() => {
-            res = usePagePerPage(limit, offset);
-        });
-        expect(res).toEqual(expected);
+        const { result } = renderHook(() => usePagePerPage(limit, offset));
+        expect(result.current).toEqual(expected);
     });
 
     it.each`
-    filter                    | apply        | selected                                       |result
+    filter                    | apply        | selected                                       |finalResult
     ${{ advisory_type: 2 }}     | ${jest.fn()} | ${[{ id: "advisory_type", chips: [{ id: 1 }] }]} | ${{ filter: { advisory_type: undefined }}}
     ${{advisory_type: [2,1]}} | ${jest.fn()} | ${[{ id: "advisory_type", chips: [{id: 1}] }]} | ${{filter: {advisory_type: [2]}}}
     ${{search: "asd"}}        | ${jest.fn()} | ${[{ id: "search"}]}                           | ${{filter: {}, search: ''}}
-    `('useRemoveFilter: should return correct filter for $filter',({filter, apply, result, selected}) => {
-        let deleteFilters;
-        testHook(() => {
-            [deleteFilters] = useRemoveFilter(filter, apply);
-        });
-        deleteFilters({}, selected)
-        expect(apply).toHaveBeenCalledWith(result);
+    `('useRemoveFilter: should return correct filter for $filter',({filter, apply, finalResult, selected}) => {
```

Comment:
Wow, never seen this method

Reply:
Perhaps we could use this variant instead https://jestjs.io/docs/api#1-describeeachtablename-fn-timeout? Seems to me more readable. But it's mine subjective opinion